data/sles4sap/qe_sap_deployment/qesap_gcp.yaml

```diff
@@ -12,7 +12,6 @@ terraform:
     region: '%REGION%'
     deployment_name: '%DEPLOYMENTNAME%'
     os_image: '%OS_VER%'
-    private_key: '%SSH_KEY_PRIV%'
     public_key: '%SSH_KEY_PUB%'
```

Comment:
Is it OK that in this file and the ones below it's `%SSH_KEY_PUB%` and in the file on top it's `%SLES4SAP_SSHKEY%`? Looking at the code in Perl it looks like `%SSH_KEY_PUB%` is the "correct" one.

Reply:
yes was ok from a functional point of view (like conf.yaml for Hanasr/mr_test and for qesap_regressio are done using two different mechanism) but not ok from the point of view of code complexity. I decided to change more code to harmonize the way we compose the pub sshkey name.

More VR are running